playground/backend/internal/utils/system_utils.go

```diff
@@ -16,14 +16,59 @@
 package utils
 
 import (
+	"beam.apache.org/playground/backend/internal/environment"
+	"beam.apache.org/playground/backend/internal/logger"
+	"net/http"
+	"os"
+	"path/filepath"
 	"reflect"
 	"runtime"
 	"strings"
 )
 
+const (
+	executableFiles = "executable_files"
```

Comment:
Don't we already have this constant somewhere?

Reply:
Yes, but it is a private constant. After `[BEAM-13308]` we could use this variable like an environment variable.